src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -290,6 +293,9 @@ export class SuggestController implements IEditorContribution {
 			adjustWhitespace: !(suggestion.insertTextRules! & CompletionItemInsertTextRule.KeepWhitespace)
 		});
 
+		scrollState.restoreRelativeVerticalPositionOfCursor(this.editor);
+		this.editor.pushUndoStop();
```

Comment:
Why are you adding an undo at all?

Reply:
@jrieken 
I'm sorry, I misunderstood. I'll remove it.